autoload/cm/sources/necovim.vim

```diff
@@ -0,0 +1,12 @@
+"=============================================================================
+" FILE: necovim.vim (NCM source)
+" AUTHOR:  Shougo Matsushita <Shougo.Matsu@gmail.com>
+" License: MIT license
+"=============================================================================
+
+function! cm#sources#necovim#refresh(opt, ctx)
+  let l:startcol = necovim#get_complete_position(a:ctx.typed)
```

Comment:
Just out of curiosity: Is there a reason not to be explicit about scope except subjective preference?